docs/examples/README.md

```diff
@@ -53,16 +55,13 @@ class Prior(AbstractPrior):
     [mean](https://docs.jaxgaussianprocesses.com/api/mean_functions/)
     and [kernel](https://docs.jaxgaussianprocesses.com/api/kernels/base/) function.
 
-    A Gaussian process prior parameterised by a mean function :math:`m(\\cdot)` and a kernel
-    function :math:`k(\\cdot, \\cdot)` is given by
-
-    .. math::
-
-        p(f(\\cdot)) = \mathcal{GP}(m(\\cdot), k(\\cdot, \\cdot)).
+    A Gaussian process prior parameterised by a mean function $`m(\cdot)`$ and a kernel
+    function $`k(\cdot, \cdot)`$ is given by
+    $`p(f(\cdot)) = \mathcal{GP}(m(\cdot), k(\cdot, \cdot))`$.
```

Comment:
Why the backticks here?

Reply:
Since this is an example of a docstring, I have included backticks as they're necessary for maths in docstrings to be formatted correctly.